source/MaterialXRuntime/Commands/PrimCommands.cpp

```diff
@@ -35,6 +35,27 @@ void createPrim(RtStagePtr stage, const RtToken& typeName, const RtPath& path, R
 
 void createPrim(RtStagePtr stage, const RtToken& typeName, const RtPath& parentPath, const RtToken& name, RtCommandResult& result)
 {
+    static RtTokenSet LOOK_MANAGEMENT_NODES { RtToken("collection"),
+                                              RtToken("look"),
+                                              RtToken("lookgroup"),
+                                              RtToken("materialassign")
+                                            };
+    static RtTokenSet MATERIAL_NODES { RtToken("surfacematerial"),
```

Comment:
Do we need special treatment for material nodes? Aren't these just nodes with a nodedef as any other node?

Reply:
This was added since we don't support material nodes within graphs as material elements could never. I'm not sure what will happen so wanted to be safe first.